.circleci/config.yml

```diff
@@ -0,0 +1,164 @@
+# fun packaging CI
+version: 2
+
+# List jobs that will be integrated and executed in a workflow
+jobs:
+
+  # Build fun application package
```

Comment:
copy/paste fail?

Build xblock package?

Reply:
No i tried to make configuration generic. It's a FUN application. WDYT?

Reply:
Maybe here just "Build application package" is better.